sdks/go/pkg/beam/core/runtime/exec/pardo.go

```diff
@@ -144,12 +145,13 @@ func (n *ParDo) processMainInput(mainIn *MainInput) error {
 	// is that either there is a single window or the function doesn't observe windows, so we can
 	// optimize it by treating all windows as a single one.
 	if !mustExplodeWindows(n.inv.fn, elm, len(n.Side) > 0) {
-		return n.processSingleWindow(mainIn)
+		_, processResult := n.processSingleWindow(mainIn)
```

Comment:
Please add comments to both these calls to make it clear that they are ignored because only SDFs should have process continuations.

Reply:
Done